packages/spaces/commands/info.js

```diff
@@ -14,7 +14,11 @@ async function run(context, heroku) {
 
   let space = await heroku.get(`/spaces/${spaceName}`, {headers})
   if (space.state === 'allocated') {
-    space.outbound_ips = await heroku.get(`/spaces/${spaceName}/nat`)
+    try {
+      space.outbound_ips = await heroku.get(`/spaces/${spaceName}/nat`)
+    } catch (error) {
+      cli.warn(`Retrieving NAT details for the space failed with ${error}`)
```

Comment:
Under what condition do we expect this error message to be relevant? I suspect it's not all possible errors.

Reply:
This API call , essentially shows the Outbound IP's, and I think if that API fails , we can just show all other relevant information.  Do you want to convert the error to debug from warn ?

Reply:
Change was made here as recommeded by @ryandagg